types/lhr/settings.d.ts

```diff
@@ -69,6 +77,8 @@ export type ScreenEmulationSettings = {
   gatherMode?: boolean | string;
   /** Flag indicating that the browser storage should not be reset for the audit. */
   disableStorageReset?: boolean;
+  /** Flag indicating which kinds of browser storage should be reset for the audit. */
+  resetOriginStorage?: StorageType[]
```

Comment:
The relationship between these two properties isn't entirely clear. WDYT about renaming this to `clearAdditionalStorageTypes`?

Reply:
I like this name better if we decide to go with your previous comment https://github.com/GoogleChrome/lighthouse/pull/15508#discussion_r1344715635

Reply:
Outdated, see https://github.com/GoogleChrome/lighthouse/pull/15508#discussion_r1349337643